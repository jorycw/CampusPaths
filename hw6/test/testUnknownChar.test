# Simple test of breadth-first search algorithm.
# Verifies that it produces the correct output when either
# the first, the second or when both of the characters are 
# not in the graph

CreateGraph characters
AddNode characters A
AddNode characters B

FindPath characters A C

FindPath characters C B

FindPath characters D E