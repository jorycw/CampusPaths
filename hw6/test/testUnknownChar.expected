# Simple test of breadth-first search algorithm.
# Verifies that it produces the correct output when either
# the first, the second or when both of the characters are
# not in the graph

created graph characters
added node A to characters
added node B to characters

unknown character C

unknown character C

unknown character D
unknown character E